src/components/Pricing/Test/PlanContent.tsx

```diff
@@ -0,0 +1,192 @@
+            setRegion('eu')
+        }
+        fetch(`/api/signup-count`)
+            .then((res) => res.json())
+            .then((count) => {
+                setSignupCountToday(count)
+                setSignupCountLoading(false)
+            })
+            .catch((err) => {
+                console.error(err)
+                setSignupCountLoading(false)
+            })
+    }, [])
+
+    return (
+        <>
+            <Modal open={modalOpen} setOpen={setModalOpen}>
+                <div className="px-5">
+                    <div className="max-w-4xl mx-auto mt-12 relative rounded-md overflow-hidden">
+                        <iframe
+                            className="m-0"
+                            width="100%"
+                            height="400"
+                            src="https://app.posthog.com/embedded/gQMqaRP0ZH0V3P3XXrSDnNcqDGoe7Q?refresh=true"
+                        />
+                    </div>
+                </div>
+            </Modal>
+            {activePlan === 'free' ? (
+                <FreePlanContent onFreeTierClick={onFreeTierClick} />
+            ) : (
+                <PaidPlanContent onFreeTierClick={onFreeTierClick} />
+            )}
+            <div className="mt-auto">
+                <div className="mb-4">
+                    <label className="block opacity-75 text-[15px] mb-1">Cloud region</label>
+                    <div className="flex gap-2">
+                        <RegionButton active={region === 'us'} onClick={() => setRegion('us')}>
+                            US (Virginia)
+                        </RegionButton>
+                        <RegionButton active={region === 'eu'} onClick={() => setRegion('eu')}>
+                            EU (Frankfurt)
+                        </RegionButton>
+                    </div>
+                </div>
+                <TrackedCTA
+                    event={{
+                        name: `clicked Get started - free`,
+                        type: 'cloud',
+                        intent: activePlan,
+                    }}
+                    size="lg"
+                    width="full"
+                    className="shadow-md !block"
+                    to={`https://${region === 'eu' ? 'eu' : 'app'}.posthog.com/signup`}
+                >
+                    Get started - free
+                </TrackedCTA>
+                <p
+                    className={`text-sm text-center mt-4 mb-0 transition-opacity text-primary/75 dark:text-primary-dark/75 ${
+                        signupCoundLoading ? 'opacity-0' : 'opacity-100'
+                    }`}
+                >
+                    {signupCountToday ? <strong>{signupCountToday}</strong> : 'Tons of'} companies signed up{' '}
+                    {signupCountToday ? (
+                        <button className="font-bold text-red dark:text-yellow" onClick={() => setModalOpen(true)}>
+                            today
+                        </button>
```

Comment:
When I clicked this button the page dimmed (as if a modal were supposed to show up) but then took forever to actually load the insight. Need a loading state. Not needed for a test of this page, but should fix it we decide to roll with it.